### DISCUSSION TITLE
Time difference between sending and receiving mail

### DISCUSSION THREAD

**Discussion by gulsenkeskin** [chatwoot/chatwoot#4487] [Q&A] [answered]
The mails I send from Chatwoot go quickly, but the mails I send from Gmail arrive late to chatwoot. How do I avoid this problem?

**Comment by sojan-official** [comment, marked as answer]
Previously for SMTP/IMAP mailboxes, Chatwoot used to fetch emails at a 5-minute interval; from `v2.4.0`, this configuration has been changed to a 1-minute gap. This should solve the issue.
